### ISSUE TITLE
Fixes to make G::V's example sms code work:

### ISSUE THREAD

**Issue by alansz** [tempire/perl-google-voice#13]
In Google/Voice.pm and Google/Voice/Feed.pm, 'attrs' should now be 'attr'

In Google/Voice.pm, Mojo::JSON->new..->decode should now be Mojo::JSON::decode_json

IO::Socket::SSL 1.96 or higher is required (but G::V will install without it, and should instead fail with a dependency)

**Comment by tempire** [comment]
Indeed. If you make a pull request, and it passes tests, I'll update the module.

**Comment by brandonwbrown** [comment]
This seems to work, or at least get past the errors, but then I get this: 

Can't call method "sms" on an undefined value at ./srd_download_hl7.pl line 131.

Since there is no error logging I guessed that the username might not require \@gmail.com but then this:

Malformed JSON: Expected string, array, object, number, boolean or null at line 0, offset 0 at /Library/Perl/5.18/Google/Voice.pm line 103.

Ideas? It would be great to get this going...

**Comment by tempire** [comment]
It requires the full username, so @gmail.com is correct. It's likely that the format of html that google voice is returning has changed a bit.

You might want to try running the tests at t/voice.t, inserting your own username, password, and phone number. That will tell you what is working, and give you a start on trying to debug the changes.
